code/espurna/sensors/INA219Sensor.h

```diff
@@ -431,7 +444,10 @@ class INA219Sensor : public I2CSensor<BaseEmonSensor> {
             .bus_range = _bus_range,
         });
 
-        _port.calibration(_params.calibration);
+         corr_calibration = corr_calibration ? corr_calibration * _params.calibration : _params.calibration; 
```

Comment:
^ and would mean, we apply some sort of func that creates Parameters on the fly instead of choosing one of our hard-coded ones

Reply:
what do you mean?

Reply:
see above. right now, we hard-code calibration based on pga config in `_fromGain()`.